.gitpod/tasks/2-app-dev/init.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 
 gp sync-await dependencies-installed
-node ./webstone/packages/create-webstone-app/bin.js webstone-dev-app
+node ./webstone/packages/create-webstone-app/bin.js webstone-dev-app --type=application
```

Comment:
By introducing arguments to the command we always need to have the application name as a second param after `create-webstone-app`. In case the app should be created in the current directory, leaving it empty won't work anymore, we would need `pnpm create webstone-app . --type=plugin` with a dot as argument for the current directory

Reply:
We may want to add a check, if the first argument startsWith(--type) or something? 🤔

Reply:
Are `--` arguments treated as positional? It sounds like it, so we could make it more generic and check if an argument starts with `--` (in case we introduce more than `--type`) and if that's the case, we assume `.` for the app directory.